src/types/queries.tsx

```diff
@@ -675,9 +675,9 @@ export const StateFragmentFragmentDoc = gql`
             id
             coords: keys
             biome: value(match: { via: [{ rel: "Biome" }] })
-            bags: nodes(match: { kinds: ["Bag"], via: { rel: "Equip", dir: OUT } }) {
+            bags: nodes(match: { kinds: ["Bag", "Item"], via: { rel: "Equip", dir: OUT } }) {
```

Comment:
is that one right? ... fetching equip'd Items under the "bags" section .... does _kinda_ make sense, but looks like a an accident

Reply:
Looks like an accident, items aren't equipped directly to seekers (yet)
The other part of the query looked right
`slots: edges(match: {kinds: ["Resource", "Item"], via: {rel: "Balance"}}) {`

Reply:
We don't have an equiped section yet so they go into a bag and we treat it like a stack of 1 item. It's the only way we can see them currently

Reply:
That does look a bit off